### PR TITLE
Fixed paths to raw_dataset_saved.json and features_processed_final.csv.

### PR DESCRIPTION
**Pull request by JPBergsma**
As mentioned in PR#4 the paths to some files were incorrect. This PR should fix that.

I would not know how the students should move the file features_processed_final.csv if the notebook is run on a remote server like binder. So I applied the change I previously suggested.
Edit:(I guess you could do it with os.replace, but I do not think this is something the students should know.)

### CODE REVIEW

File: notebooks/exercise7-oqmd-optimade-tutorial.ipynb [hunk 1]
```diff
@@ -434,10 +411,10 @@
...
     "\n",
     "# Input feature data - generated using Magpie on POSCAR files\n",
-    "if not os.path.isfile(\"features_processed_final.csv\"):\n",
-    "    raise RuntimeError(\"Please complete the optional step above, or copy the example data from `oqmd_exercise_data` into this directory.\")\n",
-    "\n",
-    "features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
+    "if os.path.isfile(\"./features_processed_final.csv\"): # Case where the optional part has been done\n",
+    "    features = open(\"./features_processed_final.csv\",\"r\").read().strip().split(\"\\n\")\n",
+    "else: \n",
+    "    features = open(\"./oqmd_exercise_data/features_processed_final.csv\",\"r\").read().strip().split(\"\\n\") # When the optional part was not done\n",
```
Comment (ml-evs) [inline notebooks/exercise7-oqmd-optimade-tutorial.ipynb]: Part of the problem here is that the students wont have access even to `./oqmd_exercise_data/` if they are running with Colab.

**Review by ml-evs — CHANGES_REQUESTED** [review]
Thanks @JPBergsma, I think lots of extra changes have been bundled with this though (binary outputs etc.). Could you adjust it to just update the code please?

File: notebooks/exercise7-oqmd-optimade-tutorial.ipynb (hunk 1 above)
Comment (JPBergsma) [reply notebooks/exercise7-oqmd-optimade-tutorial.ipynb]: Would it not be better to download the files from some location if it is so difficult to read the files from a hard drive in colab? 
The other option would be to drop colab. And just use binder or let the students run the jupyter notebook localy.

**Comment by ml-evs** [comment]
Change stability to `_oqmd_stability`.

**Review by ml-evs — APPROVED** [review]
Thanks for updating it @JPBergsma, I guess its not a massive deal that the outputs have changed, so happy to just accept this now

**Comment by ml-evs** [comment]
> Thanks for updating it @JPBergsma, I guess its not a massive deal that the outputs have changed, so happy to just accept this now

Ah, in fact the outputs are fixed, thanks! Feel free to merge if you think it is ready

**Pull request merged**